### PR TITLE
changed perl6 -> raku

### PR DESCRIPTION
**Pull request by finanalyst**
nqp references to perl6 -> raku and $*PERL -> $*RAKU

### CODE REVIEW

File: .idea/misc.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Raku v2020.02" project-jdk-type="Raku SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Raku v2019.07.1" project-jdk-type="Raku SDK" />
```
Comment (lizmat) [inline .idea/misc.xml:3]: Why the going back in version here?

File: lib/CompUnit/PrecompilationRepository/Document.pm6 [hunk 2]
```diff
@@ -3,14 +3,14 @@ use nqp;
 class CompUnit::PrecompilationRepository::Document is CompUnit::PrecompilationRepository::Default {
     
     method !load-handle-for-path(CompUnit::PrecompilationUnit $unit) {
-            my $preserve_global := nqp::ifnull(nqp::gethllsym('perl6', 'GLOBAL'), Mu);
+            my $preserve_global := nqp::ifnull(nqp::gethllsym('raku', 'GLOBAL'), Mu);
```
Comment (lizmat) [inline lib/CompUnit/PrecompilationRepository/Document.pm6:6]: The name of the HLL language implementation is `'Raku'`, not `'raku'`.

File: .idea/misc.xml (hunk 1 above)
Comment (finanalyst) [reply .idea/misc.xml:3]: Thank you. That is my version of Comma, I think. The most uptodate one fails.
I will remove .idea files from being updated by git.

Comment (finanalyst) [reply .idea/misc.xml]: I added .idea to .gitignore, but this seems to delete .idea in the main repository too.
Not sure how to eliminate this inconsequential part of the commit from the main changes below.
I could close the PR and open a new one?

Comment (lizmat) [reply .idea/misc.xml]: That question is above my git pay grade, I'm afraid :-)  So opening a new one seems safest.

Comment (JJ) [reply .idea/misc.xml]: This is comma specific. The file itself should probably be eliminated.

**Pull request merged**